packages/web/src/components/tracks-table/TracksTable.tsx

```diff
@@ -410,30 +419,44 @@ export const TracksTable = ({
       const isLocked = !isFetchingNFTAccess && !hasStreamAccess
       const deleted =
         track.is_delete || track._marked_deleted || !!track.user?.is_deactivated
+      const icon = track.is_unlisted ? (
+        <IconVisibilityHidden color='subdued' size='m' />
+      ) : isContentUSDCPurchaseGated(track.stream_conditions) ? (
+        <IconLock color='subdued' size='m' />
+      ) : isContentCollectibleGated(track.stream_conditions) ? (
+        <IconCollectible color='subdued' size='m' />
+      ) : (
+        <IconSpecialAccess color='subdued' size='m' />
+      )
```

Comment:
no need to change it, but you could instead set the icon component and pass the props in once when it's rendered below

Reply:
tbh I Reed's approach here since its gives us quick access to tweaking props individually (should they ever need to differ)

Reply:
Actually they probably will need to differ if we end up adding tooltips like explaining why its locked

Reply:
yeah that's fine. If we expect them to diverge, this is better 👍